lib/raxx.ex

```diff
@@ -390,10 +390,10 @@ defmodule Raxx do
   ## Examples
 
       iex> request(:GET, "/")
-      ...> |> set_header("referer", "example.com")
```

Comment:
This is not actually incorrect. 
https://en.wikipedia.org/wiki/HTTP_referer

Reply:
Wow, learn something new every day! I'll update the PR :+1: